### PR TITLE
Adjust footer-button spec

### PR DESCRIPTION
**Pull request by anselmbradford**
## Changes

- Adjust footer-button to immediately scroll to top if requestAnimationFrame is not supported, instead of silently doing nothing.
- Adjust footer-button spec to provide full branch coverage.

## Testing

1. `gulp test:unit --specs=js/modules/footer-button-spec.js` should pass with full coverage.

## Notes

- If this still fails on Travis periodically we could increase https://github.com/cfpb/cfgov-refresh/compare/fix_button_spec?expand=1#diff-c2f27422225b9f5a39bc1f4247dc96a7R54

### CODE REVIEW

File: cfgov/unprocessed/js/modules/footer-button.js [hunk 1]
```diff
@@ -5,19 +5,14 @@
    scrolltop-animation-without-jquery
    ========================================================================== */
 
-
 // Required modules.
 const behavior = require( './util/behavior' );
```
Comment (sebworks) [inline cfgov/unprocessed/js/modules/footer-button.js]: Switch to import statement.

Comment (anselmbradford) [reply cfgov/unprocessed/js/modules/footer-button.js]: Fixed!

**Comment by sebworks** [comment]
Merge when ready.

**Comment by jimmynotjim** [comment]
At some point we should update the button to use the [broader scrollTo utility](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/unprocessed/js/modules/util/scroll.js) and remove these two files.

**Pull request merged**